pages.de/common/7zr.md

```diff
@@ -0,0 +1,17 @@
+# 7zr
+
+> Ein Dateiarchivierer mit hoher Kompressionsrate.
+> Eine alleinstehende Version von `7z`, die nur .7z Dateien unterstützt.
+> Weitere Information: <https://www.7-zip.org/>.
```

Comment:
```suggestion
> Mehr Informationen: <https://www.7-zip.org/>.
```

Reply:
We didn't get these :thinking:

Reply:
I'm trying to be consistent with the language. and I think `Mehr Informationen` is a closer translation because `Weitere Informationen` is like `Further information`